plugin.video.popcorntv/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.popcorntv"
+       name="PopcornTv"
+       version="1.0.17"
+       provider-name="Nightflyer">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
+    <import addon="script.module.beautifulsoup" version="3.0.8"/>
+    <import addon="plugin.video.youtube" version="5.1.16"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource"
+            library="default.py">
+        <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <language>it</language>
+    <platform>all</platform>
+    <summary lang="it">La web TV On-Demand, legale e gratuita. Cinema, Musica, Sport e tanto di più</summary>
```

Comment:
it_IT

Reply:
@enen92 What about the language tag? "it" is fine there or have I to change it there too.

Reply:
@enen92 What about the language tag? "it" is fine there or have I to change it there too.

Reply:
Language tag still uses the old format :) keep it as it